distributed/distributed.yaml

```diff
@@ -129,6 +129,7 @@ distributed:
   dashboard:
     link: "{scheme}://{host}:{port}/status"
     export-tool: False
+    graph_max_items: 500  # maximum number of tasks to try to plot in graph view
```

Comment:
The config generally uses hyphens rather than underscores by convention.

Reply:
I recommend 5000 as a default value.  Hopefully we can set it to something smaller during a test.  Any thoughts on how best to test this?

Reply:
Yes, the manner in which this caused current tests to fail is illustrative :)